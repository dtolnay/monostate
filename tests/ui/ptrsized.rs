use monostate::MustBe;

fn main() {
    let _ = MustBe!(1usize);
    let _ = MustBe!(1isize);
}
