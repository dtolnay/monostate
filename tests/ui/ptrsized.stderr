error: serde data model only uses consistently sized integer types, not usize
 --> tests/ui/ptrsized.rs:4:21
  |
4 |     let _ = MustBe!(1usize);
  |                     ^^^^^^

error: serde data model only uses consistently sized integer types, not isize
 --> tests/ui/ptrsized.rs:5:21
  |
5 |     let _ = MustBe!(1isize);
  |                     ^^^^^^
